envs/example/defaults.yml

```diff
@@ -40,6 +40,11 @@ memcached:
   user: nobody
   max_connections: 1024
 
+swift:
+  enabled: False
+  haproxy_vip: "{{ fqdn }}"
```

Comment:
Okay, that's fair that the site would override it, but I think usually we define a top level swift_fqdn, and feed that into the haproxy_vip.